### PR TITLE
Add a few tests for letra.js and options button

### PR DESCRIPTION
**Pull request by tbarland77**
I got bored and ended up writing a few tests already for #124. :joy: Noticed some interesting things while doing this... Looks like the code coverage reporter is not picking up anything in the components folder for the client. After doing some poking around jest appears to be ignoring these files because they have no methods.

Which leads me to my next question. Do you see value in template tests @jayehernandez? Almost all the tests I wrote for the options button involve the template. Checking to make sure a css class is there, testing a button click event, etc. These tests won't get reflected in the code coverage but I think they could still be useful tests. Was curious what your opinion on it might be?

### CODE REVIEW

**Comment by jayehernandez** [comment]
I do think there's still value in testing the templates! Mostly just to make sure that everything is still working the way that it's supposed to. As for Jest not covering these, it's probably okay for now.

File: server/test/letra.test.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+});
+
+describe('/languages', () => {
+    it('returns a 200 response', async () => {
+      const res = await request(app).get('/languages');
+      expect(res.statusCode).toEqual(200);
+    });
+    it('returns languages as the response body', async () => {
+      const res = await request(app).get('/languages');
+      console.log(res);
```
Comment (jayehernandez) [inline server/test/letra.test.js:24]: Is there a reason for the `console.log` here? :)

Comment (tbarland77) [reply server/test/letra.test.js]: Good catch. Looks like I had forgotten to remove those earlier. :sweat_smile: They should be removed now.

Comment (jayehernandez) [reply server/test/letra.test.js]: Thank you!!

**Pull request merged**